scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -48,6 +48,12 @@ <h3 ng-show="desk.edit._id" translate>Edit "{{desk.edit.name}}" Desk</h3>
                                 </div>
                             </div>
                             <div class="field">
+                                <label translate>Desk language</label>
+                                <input type="text" class="line-input fullwidth-input"
```

Comment:
yep, maybe you are right. Should I display all languages selected there, or just languages that are source (have setted source as true)?